New_wave3.rb

```diff
@@ -0,0 +1,104 @@
+#Create Plant class
+class Planet
+  attr_reader :planet,:desc,:orbit,:diameter,:rotate,:distance
+  def initialize(planet,desc,orbit,diameter,rotate,distance)
+    @planet = planet
+    @desc = desc
+    @orbit =orbit
+    @diameter=diameter
+    @rotate = rotate
+    @distance = distance
+  end
+  def planet_attributes
+    attributes = "Planet: #{@planet}\nDesc: #{@desc}\nOrbit: #{@orbit}\nDiameter: #{@diameter}\nYear Length: #{@rotate}\nDistance from Sun: #{@distance}\n"
+    return attributes
+  end
+end
+
+#Create Solar System Class
+class SolarSystem
+  attr_accessor :planet
```

Comment:
Why make `planet` an accessor?  Why not a reader?  Do you have a reason users would be able to change `planet`?